README.md

```diff
@@ -12,13 +12,37 @@ Stratus is an EVM executor and JSON-RPC server with custom storage that scales h
 Check our `CONTRIBUTING.md` to get started.
 
 ## Getting Started with Stratus
+To run the optimized version of Stratus, ensure you have installed the dependencies:
 
-> Ensure you have installed the dependencies
+- [Rust](https://www.rust-lang.org/tools/install)
+- [just](https://github.com/casey/just)
 
-- Rust
-- Just system
-- Node.js and NPM
+Then simply run:
+
+```bash
+just run-release
+```
+
+### Testing
+
+To run tests, you also need to:
+
+- Install Git
+- Install [asdf version manager](https://asdf-vm.com/) and use it to install
```

Comment:
`just setup` should automatically install `asdf` and the desired versions of Node and Solidity.